changelogs/unreleased/gh-8598-netbox-stream-space-__index-bug.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/box
+
+* Fixed a bug in the `net.box` stream object `space` table `__index` metamethod
+  that didn't allow accessing spaces with `_stream` and `_stream_space_cache`
+  names (gh-8598).
```

Comment:
```suggestion
* Fixed a bug in the `__index` metamethod of the `space` table:  
it was impossible to access spaces named `_stream` and `_stream_space_cache` 
using the `net.box` stream object (gh-8598).
```

Reply:
> Fixed a bug in the `__index` metamethod of the `space` table:

What `space` table are we talking about?